.github/workflows/ci.yaml

```diff
@@ -6,32 +6,10 @@ on:
   pull_request:
 
 jobs:
-  setup:
-    runs-on: ubuntu-latest
-    outputs:
-      strategy: ${{steps.load.outputs.strategy}}
-
-    steps:
-      - uses: actions/checkout@v3
-      - id: load
-        run: echo "strategy=$(echo $(cat strategy.json))" >> $GITHUB_OUTPUT
-
   canary:
-    needs: [setup]
-    strategy: ${{fromJson(needs.setup.outputs.strategy)}}
-    runs-on: ${{matrix.os}}
-
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          # Choose colcon-notification for a canary build. It has colcon
-          # dependencies and debian patches, so exercieses a fair amount of the
-          # CI action features.
-          repository: colcon/colcon-notification
-      - uses: actions/setup-python@v4
-        with:
-          python-version: ${{matrix.python}}
-      - uses: actions/checkout@v3
-        with:
-          path: colcon_ci
-      - uses: ./colcon_ci/
+    uses: ./.github/workflows/pytest.yaml
```

Comment:
In other repositories, this will be `colcon/ci/.github/workflows/pytest.yaml@main`
https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow